scripts/test.ps1

```diff
@@ -1,15 +1,6 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-$ErrorActionPreference = 'Stop'
+Write-Host "No unittests defined for this repo."
```

Comment:
This reminds me: we do have unittests in https://github.com/microsoft/QuantumKatas/tree/master/utilities/DumpUnitaryTest. Are we skipping them today? It would be nice to add that validation

Reply:
We are indeed skipping those tests, and I think it would be nice to add them. `dotnet test utilities\DumpUnitaryTest` runs the test.

Reply:
Included the DumpUnitary sln to build/tests.
For good measure, I've also added the Katas.sln to the list of tests, even though there are no unittests right now. Just to be consistent that all we build is also targeted for testing.